locales/help_enu/gkhIntro.html

```diff
@@ -1,57 +1,64 @@
-<link rel="stylesheet" href="styles.css" type="text/css" />
-<title>Introduction</title>
+    <meta charset="utf-8" />
+    <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
+    <link rel="stylesheet" href="styles.css" type="text/css" />
+    <title>Introduction</title>
 </head>
 <body>
-<h1>Introduction</h1>
-<p>There are three principal factor lying in the base of Genealogy:</p>
-<ol>
-<li>A man's name (who?)</li>
-<li>Location, where an event occurred (where?)</li>
-<li>Date of an event (when?)</li>
-</ol> 
-<p>Using GEDKeeper you can process huge amount of genealogical data.
-The main property here is that GEDKeeper honors that three main
-factors. For example, one can quickly find all namesakes, or all men
-from the same village, or everyone who lived in a specific period of
-time. User can filter data using all the three factors. For example, to
-select namesakes, who lived in the same village in the specified
-year.</p>
-<p>User collects information (create database) using different sources
-(including archives of all kinds). The following are two main sources:
-Audit tales (audits and censuses made on more earlier or later time)
-and registers of births. Other sources are also possible, but we take
-the two sources as a basic.</p>
-<p>A database filled with entries from sources, may be counted as
-conditionally complete one. For example, we got data defining all
-citizens of a specific locality at a specific period of time, if we had
-used audits (censuses). Later we were adding more and more new
-information, and finally we got a huge data array with many people
-living in several locations during different time. Such database is
-very useful for a genealogist, because no one knows about relations
-between men in the database beforehand. This becomes more obviously
-later, after inspecting other sources, for example, registers of
-births.</p>
-<p>Why is database called conditionally complete? Because it does not
-contain strictly accurate data. Many data are approximate, or even
-estimated. For example, a date in database can be exact
-(1&nbsp;954/06/12), or much less exact (1&nbsp;954), or approximate
-(around 1&nbsp;976), or estimated (before 1&nbsp;956 or after
-1&nbsp;873). It is very important for a software to handle such dates
-correctly.</p>
-<p>GEDKeeper has a very efficient tool -- &quot;Timeline&quot;. This
-tool can make time samples for events. Using &quot;Timeline&quot; one
-can get a model of the original source -- an archival document. For
-example, after we have made a time sample for 1&nbsp;858 using
-locations of events, we can get a list similar to the original source
--- people who lived in a specific village in 1&nbsp;858 (as in tenth
-audit of 1&nbsp;858). This allows us to check correction of all the
-data we have added to database.</p>
-<hr/>
-<p>.</p>
+    <h1>Introduction</h1>
+    <p>There are three principal factors the basics of Genealogy:</p>
```

Comment:
Is this sentence correct for the English language?  I'm asking because had I wrote this using the Russian language I'd put \`\`in'' after \`\`factors'', or place dash at least.